api/client/set.go

```diff
@@ -0,0 +1,67 @@
+package client
+
+import (
+	"fmt"
+
+	flag "github.com/docker/docker/pkg/mflag"
+	"github.com/docker/docker/pkg/units"
+	"github.com/docker/docker/runconfig"
+)
+
+func (cli *DockerCli) CmdSet(args ...string) error {
+	cmd := cli.Subcmd("set", "CONTAINER [CONTAINER...]", "Setup/Update resource configs for a container", true)
+	flCpusetCpus := cmd.String([]string{"#-cpuset", "-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
+	flCpusetMems := cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
+	flCpuShares := cmd.Int64([]string{"c", "-cpu-shares"}, 0, "CPU shares (relative weight)")
+	flMemoryString := cmd.String([]string{"m", "-memory"}, "", "Memory limit")
+	flMemorySwap := cmd.String([]string{"-memory-swap"}, "", "Total memory (memory + swap), '-1' to disable swap")
+	flCpuQuota := cmd.Int64([]string{"-cpu-quota"}, 0, "Limit the CPU CFS (Completely Fair Scheduler) quota")
+	cmd.Require(flag.Min, 1)
+
+	cmd.ParseFlags(args, true)
+
+	var flMemory int64
+	if *flMemoryString != "" {
+		parsedMemory, err := units.RAMInBytes(*flMemoryString)
+		if err != nil {
+			return err
+		}
+		flMemory = parsedMemory
+	}
+
+	var MemorySwap int64
+	if *flMemorySwap != "" {
+		if *flMemorySwap == "-1" {
+			MemorySwap = -1
+		} else {
+			parsedMemorySwap, err := units.RAMInBytes(*flMemorySwap)
+			if err != nil {
+				return err
+			}
+			MemorySwap = parsedMemorySwap
+		}
+	}
```

Comment:
If the user doesn't specify any options what should we do?
- generate an error?
- call daemon but do nothing? current behavior
- just exit w/o calling daemon?

If nothing is going to change I don't see value in calling the daemon.  I don't have a strong opinion on this one, but I'm inclined to just exit (with zero exit code) w/o calling the daemon since technically its not an error, its just a no-op/pointless case.